README.md

```diff
@@ -73,15 +76,25 @@ This will create an install directory with the installed libraries as well as ot
 
 ## Setup
 
-The environment must be set by activating the setup script (`install/bin/setup_histfitter.sh`) in the directory where you want to work. This will create five new directories in your work directory: `analysis`, `config`, `data`, `macros`, and `results`. It also copies the `HistFactorySchema.dtd` file into the config directory. Most importantly, the setup script sets the environment variable paths that are required for the scripts to work.  The setup also adds the analysis directory in the working area to the Python path.  This makes it convenient to import additional python files without needing to install them in the build area.
+The environment must be set by activating the setup script (`install/bin/setup_histfitter.sh`) in the directory where you want to work. This will create three new directories in your work directory: `config`, `data`, and `results`. It also copies the `HistFactorySchema.dtd` file into the config directory. Most importantly, the setup script sets the environment variable paths that are required for the scripts to work. Use the `-e` flag to obtain a copy of the `analysis` and `macros` folders containing the examples, which are a good starting point for creating your own analysis. If you want to import additional python files, make sure that they are located in a folder which is added to the `PYTHONPATH` environment variable (like `analysis`).
 
 ```
+mkdir HistFitterProject
```

Comment:
This and the change below, would be more concise if described in two steps:

Make a directory to work in:
```
mkdir HistFitterProject
cd HistFitterProject
```
Then setup PATH env variable to include HistFitter and this working area.  With a local build from source, this can be done via:
`source /path/to/install/bin/setup_histfitter.sh`
 or you can also run the setup script from any directory if you use the -p flag and the path to the work directory.
 `source /path/to/install/bin/setup_histfitter.sh -p /path/to/HistFitterProject`

If you are using a StatAnalysis release, then you don't need to use the full path, but just `setup_histfitter.sh` as it is already in PATH.